exonum/src/api/mod.rs

```diff
@@ -332,9 +332,15 @@ impl ApiAggregator {
         self.inner.insert(prefix.into(), builder);
     }
 
-    fn explorer_api(shared_node_state: SharedNodeState) -> ServiceApiBuilder {
+    fn explorer_api(
+        blockchain: &Blockchain,
+        shared_node_state: SharedNodeState,
+    ) -> ServiceApiBuilder {
+        use self::node::public::ExplorerApi;
```

Comment:
We typically have all our imports at the top of the file, don't we?
cc @DarkEld3r

Reply:
We don't have a strict rule about that, but I prefer imports at the top of the file because it is more manageable in terms of refactoring.